### DISCUSSION TITLE
Welcome to MEAN stack  Pharmacy-Mangment-System Discussions!

### DISCUSSION THREAD

**Discussion by LalanaChami** [LalanaChami/Pharmacy-Mangment-System#173] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brendonenoe** [comment]
Hie, I'm Brendon Manyara ....I'm here to learn more and also contribute to wherever possible to the community

**Reply by LalanaChami** [reply]
Hi Brendon , hope you've run the project locally and played with it . If you find any interesting part that you would like to enhance or a bug you've like fix , please let me know or else just hit a PR with a brief comment. All sort of contributions are more than welcomed.

**Comment by DJALAL23** [comment]
Hi, is there any documentation for the project installation, i can't run it locally.
